pkg/agent/utils.go

```diff
@@ -259,6 +259,9 @@ func IsKubernetesVersionGe(actualVersion, version string) bool {
 
 func getCustomDataFromJSON(jsonStr string) string {
 	var customDataObj map[string]string
+	jsonStr = strings.ReplaceAll(jsonStr, "\r", "\r\n")
```

Comment:
what's this one actually fixing?

Reply:
The errors were 
`panic: invalid character '\r' in string literal`
`panic: invalid character '\n' in string literal`
`panic: invalid character '\t' in string literal`

Reply:
(I fixed `jsonStr = strings.ReplaceAll(jsonStr, "\r", "\r\n")` to `jsonStr = strings.ReplaceAll(jsonStr, "\r", "\n")` was trying to see if it just needed to be a different new line, but it wasn't happy)